config/prod/PsychENCODE-Globus-S3.yaml

```diff
@@ -0,0 +1,30 @@
+# Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html)
+template:
+  type: "http"
+  url: ""
```

Comment:
you don't need `templates/PEC-Globus-S3.j2`.  You just need to reference the [synapse-external-bucket.j2 file](https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/S3/synapse-external-bucket.j2)